sonar-csharp-plugin/src/main/resources/documentation.md

```diff
@@ -16,12 +16,27 @@ Discover and update the C#-specific [properties](/analysis/analysis-parameters/)
 
 To analyze tool-generated code (e.g. WCF code generated by `SvcUtil.exe`, protobuf code generated by `protoc`, Swagger client code generated by `NSwag`) for a specific C# project, enable the "Analyze generated code" setting inside **Project > Administration > General Settings > C#**. By default, tool-generated code files are skipped from analysis.
 
+The detection of generated code is based on file name, special comments and attributes. The currently recognized values are in [GeneratedCodeRecognizer.cs](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Helpers/GeneratedCodeRecognizer.cs).
```

Comment:
```suggestion
The detection of generated code is based on file name, special comments, and attributes. The currently recognized values are in [GeneratedCodeRecognizer.cs](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.Common/Helpers/GeneratedCodeRecognizer.cs).
```

Reply:
grammarly says there a comma needed there. @tom-howlett-sonarsource could you confirm/infirm?

Reply:
I believe Grammarly is proposing an [Oxford Comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) here. I personally wouldn't use one here, feels unnecessary.